packages/jet-brains-integration/src/web-types-generator.ts

```diff
@@ -55,49 +55,56 @@ export function getTagList(
           description: slot.description,
         };
       }),
-      events: getWebTypeEvents(component),
-      js: getJsProperties(component, options.typesSrc),
+      js: getJsProperties(component),
     };
   });
 }
 
-function getJsProperties(
-  component: Component,
-  typesSrc?: string
-): JsProperties {
+function getJsProperties(component: Component): JsProperties {
   return {
-    properties: getWebTypeProperties(component, typesSrc),
+    properties: getWebTypeProperties(component),
     events: getWebTypeEvents(component),
   };
 }
 
-function getWebTypeProperties(
-  component: Component,
-  typesSrc = "types"
-): WebTypeAttribute[] {
+/**
```

Comment:
It looks like this is just exposing non-public properties. Can we rename it to be more clear?

```
/**
 * @param field The Custom Elements Manifest class field to evaluate
 * @return public properties
 */
function isPublicProperty(field: schema.ClassField): boolean {
  return field.static !== true && (field.privacy === 'public' || field.privacy === undefined)
}
```

Reply:
Sure thing!